core/src/main/java/com/capitalone/dashboard/model/Incident.java

```diff
@@ -79,9 +79,9 @@
 
     public void setStatus(String status) { this.status = status; }
 
-    public long getClosedTime() { return closedTime; }
+    public Long getClosedTime() { return closedTime; }
 
-    public void setClosedTime(long closedTime) { this.closedTime = closedTime; }
+    public void setClosedTime(Long closedTime) { this.closedTime = closedTime; }
 
     public void setClosedTime(String closedTime) { this.closedTime = DATE_FORMATTER.parseMillis(closedTime); }
```

Comment:
public void setClosedTime(String closedTime) { this.closedTime = DATE_FORMATTER.parseMillis(closedTime); 
} 
should be written as
public void setClosedTime(String closedTime) { this.setClosedTime( DATE_FORMATTER.parseMillis(closedTime)); 
}

Reply:
@rvema : Sure. no problem. Yes, this is an existing overloaded method, and so we can make it use the other one. : ). Will change it out. Thanks.